CodingChallenges/CC_137_4D_Noise_Loop/P5/index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
+  <title>4D Open Simplex Noise Loop</title>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/p5.js/0.7.2/p5.min.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/p5.js/0.7.2/addons/p5.dom.min.js"></script>
+  <script src="OpenSimplexNoise.js"></script>
+  <script src="CCapture.all.min.js"></script>
```

Comment:
Is there a CDN link for `CCapture`? I wasn't aware of this library, great option for recording!